### PR TITLE
Add German Translation

### PR DESCRIPTION
**Pull request by kaffeekirsche**

### CODE REVIEW

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "itemGroup.refinedmachinery.refinedmachinery" : "Refined Machinery",
+  "__comment": "Resource Items",
+  "item.refinedmachinery.copper_dust" : "Kupferstaub",
+  "item.refinedmachinery.copper_ingot" : "Kupferbarren",
+  "item.refinedmachinery.lead_dust" : "Leadstaub",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: Lead -> Blei

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.lead_ingot" : "Leadbarren",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: Lead -> Blei

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.nickel_dust" : "Nickelstaub",
+  "item.refinedmachinery.nickel_ingot" : "Nickelbarren",
+  "item.refinedmachinery.silver_dust" : "Silberstaub",
+  "item.refinedmachinery.silver_ingot" : "Silberbarren",
+  "item.refinedmachinery.tin_dust" : "Zinnstaub",
+  "item.refinedmachinery.tin_ingot" : "Zinnbarren",
+  "item.refinedmachinery.coal_dust" : "Kohlestaub",
+  "item.refinedmachinery.diamond_dust" : "Diamandstaub",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: Diaman`t`, not Diaman`d`

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.iron_dust" : "Eisenstaub",
+  "item.refinedmachinery.gold_dust" : "Goldstaub",
+  "item.refinedmachinery.steel_ingot" : "Stahlbarren",
+  "item.refinedmachinery.energized_steel_ingot" : "Energie Stahlbarren",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: it's not an "energy steel ingot"

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+  "__comment2": "Items",
+  "item.refinedmachinery.quarry_recorder" : "Quarry Recorder",
+  "item.refinedmachinery.energized_sword" : "Energie Schwert",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: energy != energized

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.linker" : "Linker Tool",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: `Verbindungswerkzeug`

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.wrench": "Utility Wrench",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: `Schraubenschlüssel`

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.robotic_wings": "Robo Flügel",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: Robo`-`Flügel

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.block_exchanger": "Block Exchanger",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: this name is not translated at all

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+  "__comment3": "Resource Blocks",
+  "block.refinedmachinery.copper_block" : "Kupferblock",
+  "block.refinedmachinery.copper_ore" : "Kupfererz",
+  "block.refinedmachinery.lead_block" : "Lead Block",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: Lead -> Blei

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.lead_ore" : "Lead Ore",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: this name is not translated at all

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.nickel_block" : "Nickel Block",
+  "block.refinedmachinery.nickel_ore" : "Nickel Ore",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: this name is not translated at all

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+  "item.refinedmachinery.wrench": "Utility Wrench",
+  "item.refinedmachinery.robotic_wings": "Robo Flügel",
+  "item.refinedmachinery.block_exchanger": "Block Exchanger",
+
+  "__comment3": "Resource Blocks",
+  "block.refinedmachinery.copper_block" : "Kupferblock",
+  "block.refinedmachinery.copper_ore" : "Kupfererz",
+  "block.refinedmachinery.lead_block" : "Lead Block",
+  "block.refinedmachinery.lead_ore" : "Lead Ore",
+  "block.refinedmachinery.nickel_block" : "Nickel Block",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: should be combined into one word

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.nickel_ore" : "Nickel Ore",
+  "block.refinedmachinery.silver_block" : "Silber Block",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: should be combined into one word

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.silver_ore" : "Silber Ore",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: `ore`?

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.tin_block" : "Zinn Block",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: should be combined into one word

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.tin_ore" : "Zinn Ore",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: `ore`?

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+
+  "__comment4": "Blocks",
+  "block.refinedmachinery.energy_furnace" : "Energie Ofen",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: maybe find a better translation

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.pulverizer" : "Pulverizer",
+  "block.refinedmachinery.quarry" : "Quarry",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: needs translation

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.solar_panel_mk1" : "Sonnenkollektor MK I",
+  "block.refinedmachinery.solar_panel_mk2" : "Sonnenkollektor MK II",
+  "block.refinedmachinery.solar_panel_mk3" : "Sonnenkollektor MK III",
+  "block.refinedmachinery.glass_block" : "Glasblock",
+  "block.refinedmachinery.black_glass_block" : "Schwarzer Glasblock",
+  "block.refinedmachinery.wireless_controller" : "Wireless Energy Controller",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: needs translation

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 21]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.wireless_transmitter" : "Wireless Energy Transmitter",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: needs translation

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 22]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.lava_generator" : "Lava Generator",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: `Lava-Generator`

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 23]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.energy_charger" : "Energy Ladegerät",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: drop the `energy` prefix

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 24]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.fluid_pump" : "Flüssigkeitspumpe",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: jusst `Pumpe` would do

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 25]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.farmer" : "Farmer Block",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: -> `Ernter`, and no `block` suffix

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 26]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.mob_grinder" : "Mob Grinder",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: translation?

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 27]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.vacuum" : "Vacuum Block",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: needs translation

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 28]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.conveyor_belt" : "Förderband",
+  "block.refinedmachinery.energy_cell" : "Kardinal Energiezelle",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: `Cardinal Energy` is the energy unit, not part of the item name

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 29]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.machine_frame" : "Machinenrahmen",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: Ma`s`chine

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 30]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.energy_cable" : "Energy Transfer Kabel",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: just `Kabel` or `Energiekabel`

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.copper_tank" : "Kupfer Flüssigkeitstank",
+  "block.refinedmachinery.silver_tank" : "Silber Flüssigkeitstank",
+  "block.refinedmachinery.steel_tank" : "Stahl Flüssigkeitstank",
+  "block.refinedmachinery.coalgen" : "Kohle Generator",
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: `Kohle-Generator`

File: src/main/resources/assets/refinedmachinery/lang/de_de.json [hunk 32]
```diff
@@ -0,0 +1,67 @@
...
+  "block.refinedmachinery.disenchanter" : "Disenchanter"
```
Comment (UpcraftLP) [inline src/main/resources/assets/refinedmachinery/lang/de_de.json]: needs translation

**Review by UpcraftLP — CHANGES_REQUESTED** [review]

**Review by kaffeekirsche — COMMENTED** [review]
Fix Translation

**Review by UpcraftLP — APPROVED** [review]

**Pull request merged**